Sketch.java

```diff
@@ -8,29 +8,79 @@ public class Sketch extends PApplet {
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(300, 300);
   }
-
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
+float circleX = 150;
+float circleT = 150;
+float[] circleY = new float[25];
   public void setup() {
     background(210, 255, 173);
+     for (int i = 0; i < circleY.length; i++) {
+    circleY[i] = random(height);
   }
+}
+  
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+    background(50);
+fill(255);
+  for (int i = 0; i < circleY.length; i++) {
+    float circleX = width * i / circleY.length;
+    ellipse(circleX, circleY[i], 25, 25);
+
+    circleY[i]++;
+
+    if (circleY[i] > height) {
+      circleY[i] = 0;
+        }
+  if(keyPressed){
+      if (keyCode == UP) {
+          circleY[i]=circleY[i]+1/2;
+       
+    }
+    else if(keyCode == DOWN){
+     circleY[i]=circleY[i]+5;
+    }
+  }
+  else{
+circleY[i]=circleY[i]+2;
+  }
```

Comment:
identation here is not great.  Lines should be in line with their appropriate indent level.